src/platform/forms-system/src/js/review/submit-states/Pending.jsx

```diff
@@ -7,6 +7,14 @@ import PreSubmitSection from 'platform/forms/components/review/PreSubmitSection'
 
 export default function Pending(props) {
   const { formConfig, onBack, onSubmit } = props;
+  let ariaDescribedBy = null;
+  // If no ariaDescribedBy is passed down from form.js,
+  // a null value will properly not render the aria label.
+  if (formConfig.ariaDescribedBySubmit !== null) {
```

Comment:
I believe this value would be `undefined` for most forms, so comparing it to `null` would have `aria-describedby="undefined"` get rendered. I haven't confirmed this problem though.

Reply:
I tested this scenario by deleting the `ariaDescribedBySubmit` attribute from the form.js. When null, it will simply not render the `aria-desribedby` attribute at all.